### DISCUSSION TITLE
AuthenticationService without password (custom user validation "outside password or current providers)

### DISCUSSION THREAD

**Discussion by guidoka** [directus/directus#10530] [Archive]
Hello,

is there a possiblity to get access_token/refresh_token via

AuthenticationService.login("default", {email: login}) only with the email (no password)?

We have a custom Authentication-Service (where i got a custom jwt with challengeid and otp to check against a webservice)

Thanks!

**Comment by rijkvanzanten** [comment]
No, there's currently no way to circumvent the auth adapter usage when using the login method on the authentication service. That being said, there's an open PR that aims to implement a way to do exactly this: https://github.com/directus/directus/pull/10375

**Reply by guidoka** [reply]
thanks a lot, i'll check that out!
